syntax = "proto3";

service Registration{
 rpc GetServerResponse(Message) returns (MessageResponse) {}
}

message Message{
 string email = 1;
 string password = 2;
 string user_type= 3;
 string phone_num = 4;
 bool confirmed = 5;
}

message MessageResponse{
 bool received = 1;
}